guides/doc-Upgrading_and_Updating/topics/con_upgrading-prerequisites.adoc

```diff
@@ -0,0 +1,28 @@
+[[upgrading_prerequisites]]
+== Prerequisites
+
+Upgrading to {Project} {ProjectVersion} affects your entire {Project} infrastructure.
+Before proceeding, complete the following:
+
+* Read the {ProjectName} {ProjectVersion} {ReleaseNotesURL}[Release Notes].
+* Plan your upgrade path.
+For more information, see xref:upgrade_paths[].
+* Plan for the required downtime. {Project} services are shut down during the upgrade.
+The upgrade process duration might vary depending on your hardware configuration, network speed, and the amount of data that is stored on the server.
++
+Upgrading {Project} takes approximately 1{range}2 hours.
```

Comment:
I'm still debating if this is correct. For vanilla Foreman (without Katello) I think 10 to 30 minutes is closer, but hard to say for sure.